truss/templates/custom_python_dx/my_model.py

```diff
@@ -0,0 +1,28 @@
+"""
+The `Model` class is an interface between the ML model that you're packaging and the model
+server that you're running it on.
+
+The main method to implement here is:
+* `predict`: runs every time the model server is called. Include any logic for model
+  inference and return the model output.
+
+See https://truss.baseten.co/quickstart for more.
+"""
+
+import truss_chains as baseten
+
+
+class Model(baseten.ModelBase):
+    # Configure resources for your model here.
+    remote_config: baseten.RemoteConfig = baseten.RemoteConfig(name="{{ MODEL_NAME }}")
```

Comment:
This template string is replaced at `init` time, very open to other suggestions here